src/types/point.js

```diff
@@ -15,11 +15,19 @@ export default class PointAnnotation extends Element {
 
   draw(ctx) {
     const options = this.options;
+    const borderWidth = options.borderWidth;
     ctx.save();
     ctx.fillStyle = options.backgroundColor;
-    setBorderStyle(ctx, options);
+    setShadowStyle(ctx, options);
+    const stroke = setBorderStyle(ctx, options);
+    options.borderWidth = 0;
     drawPoint(ctx, options, this.x, this.y);
```

Comment:
Might be worth having a test to ensure that `borderWidth = 0` correctly skips in this helper. We could add a test in this package to know if that ever breaks when upgrading the chart.js version. An alternative is to add a test in the main chart.js repo about this case

Reply:
I think it's better to add a test here. Let me implement it.

Reply:
I have added 2 test cases for point annotation with `borderWidth: 0` (1 with shadow and 1 without). If something will change in Chart.js, breaking the plugin, we should be able to recognize it